tests/pyupgrade_test.py

```diff
@@ -1092,6 +1092,10 @@ def test_fix_six(s, expected):
             'class C(object, metaclass=ABCMeta): pass',
             'class C(metaclass=ABCMeta): pass',
         ),
+        (
+            'class C(six.Iterator, metaclass=ABCMeta): pass',
```

Comment:
should also handle `from six import Iterator`  `class C(Iterator, ...):`

I wonder what happens if

```python
class C(Iterator, object):
```

🤔

Reply:
Good point, let me try to handle those cases